src/app/pages/data-protection/cloud-backup/cloud-backup-form/cloud-backup-form.component.html

```diff
@@ -0,0 +1,99 @@
+<ix-modal-header2 [requiredRoles]="requiredRoles" [title]="title" [loading]="isLoading"></ix-modal-header2>
+<mat-card>
+  <mat-card-content>
+    <form class="ix-form-container" [formGroup]="form" (submit)="onSubmit()">
+      <div class="fieldsets">
+        <ix-fieldset
+          class="fieldset"
+          [title]="'Local' | translate"
+        >
+          <ix-explorer
+            formControlName="path"
+            [label]="'Source Path' | translate"
+            [tooltip]="'Select the directories or files to be sent to the cloud for backup.' | translate"
```

Comment:
would be good to extract tooltips as we do for other forms
example `[tooltip]="helptext.ui_certificate.tooltip | translate"`